File: CS/SyncWithOutlook/CustomObjects.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace SyncWithOutlook {
    #region #customappointment
    public class CustomAppointment {
        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }
        public string Subject { get; set; }
        public int Status { get; set; }
        public string Description { get; set; }
        public int Label { get; set; }
        public string Location { get; set; }
        public bool AllDay { get; set; }
        public int EventType { get; set; }
        public string RecurrenceInfo { get; set; }
        public string ReminderInfo { get; set; }
        public object OwnerId { get; set; }
        public object OutlookID { get; set; }

        public CustomAppointment() {
        }
    }
    #endregion  #customappointment

    #region #customresource
    public class CustomResource
    {
        public string Name { get; set; }
        public int ResID { get; set; }
        public System.Drawing.Color ResColor { get; set; }

        public CustomResource()
        {
        }
    }
    #endregion #customresource
}

File: VB/SyncWithOutlook/CustomObjects.vb
Namespace SyncWithOutlook

'#Region "#customappointment"
    Public Class CustomAppointment

        Public Property StartTime As Date

        Public Property EndTime As Date

        Public Property Subject As String

        Public Property Status As Integer

        Public Property Description As String

        Public Property Label As Integer

        Public Property Location As String

        Public Property AllDay As Boolean

        Public Property EventType As Integer

        Public Property RecurrenceInfo As String

        Public Property ReminderInfo As String

        Public Property OwnerId As Object

        Public Property OutlookID As Object

        Public Sub New()
        End Sub
    End Class

'#End Region  ' #customappointment
'#Region "#customresource"
    Public Class CustomResource

        Public Property Name As String

        Public Property ResID As Integer

        Public Property ResColor As System.Drawing.Color

        Public Sub New()
        End Sub
    End Class
'#End Region  ' #customresource
End Namespace
